<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DMBEMETablePanel</class>
 <widget class="QWidget" name="DMBEMETablePanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DMBEMETablePanel</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Initialize EME Table</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="_btnSelectAll">
       <property name="text">
        <string>SelectAll</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="_btnSelectNone">
       <property name="text">
        <string>SelectNone</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="_cbASC">
       <property name="text">
        <string>ASC</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="_cbWS">
       <property name="text">
        <string>WS</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="_cbCS">
       <property name="text">
        <string>CS</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="_cbDust">
       <property name="text">
        <string>DUST</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="_cbSQM">
       <property name="text">
        <string>SQM</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="_cbDIMM">
       <property name="text">
        <string>DIMM</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
